ts/utils/card.ts

```diff
@@ -12,3 +13,51 @@ export const cardIcons: { [key in PaymentNetworkEnum]: any } = {
 export const getCardIconFromPaymentNetwork = (
   brand: PaymentNetworkEnum | undefined
 ) => fromNullable(brand).fold(defaultCardIcon, b => cardIcons[b]);
+
+const sumNumbers = (sum: number, current: number): number => sum + current;
+
+const MIN_CARD_LENGTH = 12; // The card with the shortest length is the 12-digit Maestro UK
+const MAX_AMEX_CARD_LENGTH = 15;
+const isAmexCard = /^3[47][0-9]{13}$/;
+
+export const luhnValidateCCN = (creditCard: string): boolean => {
```

Comment:
May you please write some unit tests on this specific script?

Reply:
Added required isolation tests on luhn functionality as suggested ✅